test/truffle/ecosystem/rails-app.sh

```diff
@@ -5,9 +5,7 @@ set -x
 
 cd ../jruby-truffle-gem-test-pack/gem-testing/rails-app
 
-JRUBY_BIN=../../../jruby/bin
-JRUBY=$JRUBY_BIN/jruby
-JTR=$JRUBY_BIN/jruby-truffle-tool
+JTR=../../../truffleruby/bin/jruby-truffle-tool
```

Comment:
yeah that's slightly better, but I am leaving the assumption that the jruby-truffle-gem-test-pack repo is its sibling